documentation/docs/ui/channel_list/customizations/creating_custom_view_holder_factory.md

```diff
@@ -0,0 +1,111 @@
+---
+id: uiChannelListCreatingCustomViewHolderFactory
+title: Creating Custom View Holder Factory
+sidebar_position: 3
+---
+`ChannelListView` provides a mechanism for completely changing the default view holders, as well as introducing different types of views. All you need to do is to provide your own `ChannelListItemViewHolderFactory`. Let's make an example that displays the channel's photo, name, and member count:
+
+![customer view holder](/img/custom_view_holder.png)
```

Comment:
NIT: What do you think about renaming the image to `channel_custom_view_holder`, so it will be easier to find it later? We are also going to have custom view holders in the next section.